book/modules/appendix1.tex

```diff
@@ -246,7 +283,7 @@
 
 	By adding and subtracting rows, we ``reduced'' the number of variables from some equations until
 	they were easy to solve. As an added benefit, every system along the way to System \eqref{EQEQUIVSYS}
-	was nicely organised and formatted. In fact, the systems are so well organized that we can
```

Comment:
"Organised" and "organized" on the same line.

Reply:
Welcome, American spellings :-D!